internal/utils/naming/naming.go

```diff
@@ -0,0 +1,12 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package naming
+
+import "k8s.io/apimachinery/pkg/types"
+
+func ServiceName(nn types.NamespacedName) string {
+	return nn.Name + "." + nn.Namespace
```

Comment:
should this be `namespace/name`, now that his how we are naming things ?

Reply:
align to what istio does, also same as the fqdn format in Kubernetes, e.g. `nginx.default`, `foo.bar`